food.js

```diff
@@ -0,0 +1,131 @@
+
+//I used the 'append child' method to CREATE a div for each loop-through, just in case. It seems like better practice for future functionality. Declared out here so I can use them for both cats and dogs.
+var makeMasterdogDiv = function() {
+  let masterDiv = document.createElement("div");
+  masterDiv.className = "masterdiv col-xs-6";
+  dogContentDiv.appendChild(masterDiv);
+  return masterDiv;
+}
+
+var makeMastercatDiv = function() {
+  let masterDiv = document.createElement("div");
+  masterDiv.className = "masterdiv col-xs-6 cat";
+  catContentDiv.appendChild(masterDiv);
+  return masterDiv;
+}
+var makeTypeDiv = function() {
+  let typeDiv = document.createElement("div");
+  typeDiv.className = "typeDiv";
+  masterDiv.appendChild(typeDiv);
+  return typeDiv;
+}
+//div for breeds, only used during cat loop. to make all the breeds appear in a row.
+var makeBreedDiv = function() {
+  let breedDiv = document.createElement("p");
+  breedDiv.className = "breedDiv";
+  masterDiv.appendChild(breedDiv);
+  breedDiv.innerHTML = `ideal for breeds: `
+  return breedDiv;
+}
+
+var makeVolumeDiv = function() {
+  let volumeDiv = document.createElement("p");
+  volumeDiv.className = "volumeDiv";
+  typeDiv.appendChild(volumeDiv);
+  return volumeDiv;
+} 
+//====================
+//function that runs when DOG json is loaded: a series of nested forEach loops. Probably rather a time-consuming method since each loop and subloop and sub-subloop drills top-down through the json each time. 
+//====================
+var createdogcard = function() {
+  console.log("dog food loaded");
+  //parse dog food object.
+  var dogFood = JSON.parse(dogFoodRequest.responseText);
+  console.log("dog food object", dogFood );
+  
+  //loop one: create master div with brand title.
+  dogFood.dog_brands.forEach(function(brand, brandindex){
+    masterDiv = makeMasterdogDiv();
+    console.log("brand of dog food", dogFood.dog_brands[brandindex].name );
+    masterDiv.innerHTML = `<h4 class = "brandname">${dogFood.dog_brands[brandindex].name}</h4>`;
+  
+    //loop two (nested): create type div with title.
+    dogFood.dog_brands[brandindex].types.forEach(function(type, typeindex) {
+      typeDiv = makeTypeDiv();
+      console.log("type of dog food", dogFood.dog_brands[brandindex].types[typeindex].type);
+      typeDiv.innerHTML = `<p class="foodtype">${dogFood.dog_brands[brandindex].types[typeindex].type}</p>`
+      
+      //loop three (nested): create volume/price div function. 
+      dogFood.dog_brands[brandindex].types[typeindex].volumes.forEach(function(volume, volumeindex){
+        volumeDiv = makeVolumeDiv();
+        console.log("volume name", dogFood.dog_brands[brandindex].types[typeindex].volumes[volumeindex].name );
+        volumeDiv.innerHTML = `Size: ${dogFood.dog_brands[brandindex].types[typeindex].volumes[volumeindex].name};  Price: ${dogFood.dog_brands[brandindex].types[typeindex].volumes[volumeindex].price}`
```

Comment:
...and here use ```volume.name``` & ```volume.price```.

No need to even worry about the indexes at all, since forEach returns the element at that point in the loop already as the first argument.